api/src/services/opportunities_v1/get_opportunity.py

```diff
@@ -31,7 +34,30 @@ def _fetch_opportunity(
 
 
 def get_opportunity(db_session: db.Session, opportunity_id: int) -> Opportunity:
-    return _fetch_opportunity(db_session, opportunity_id, load_all_opportunity_summaries=False)
+    opportunity = _fetch_opportunity(
+        db_session, opportunity_id, load_all_opportunity_summaries=False
+    )
+    opportunity_copy = copy.deepcopy(opportunity)
+
+    s3_config = S3Config()
+    s3_config.s3_endpoint_url = "http://localstack:4566"
+    s3_client = get_s3_client(
+        s3_config, boto3.Session(aws_access_key_id="NO_CREDS", aws_secret_access_key="NO_CREDS")
+    )
+
+    for opp_attachment in opportunity_copy.opportunity_attachments:
+        file_loc = opp_attachment.file_location
+        object_key = file_loc.split(f"{s3_config.s3_opportunity_bucket}/")[
```

Comment:
We have a file_util which has a split function for s3 urls: https://github.com/HHS/simpler-grants-gov/blob/main/api/src/util/file_util.py#L18 - which handles getting the bucket + key to pass into the next function.

While the s3 config here has a bucket configured, we should always just use whatever is in the object itself.

Reply:
great!